shifthelper/tools/__init__.py

```diff
@@ -30,11 +31,20 @@
        wait_exponential_multiplier=100,  # wait 2^i * 100 ms, on the i-th retry
        wait_exponential_max=1000,  # but wait 1 second per try maximum
        )
+@cached(cache=TTLCache(1, ttl=30))
 def get_alerts():
     alerts = requests.get(config['webservice']['post-url'])
     return alerts.json()
 
 
+def is_alert_acknowledged(uuid):
+    alerts = get_alerts()
```

Comment:
Ah right .. so if we can't even get any alerts ... the answer is still `False`...